website/shortlink/models.py

```diff
@@ -0,0 +1,33 @@
+from django.conf import settings
+from django.core import validators
+from django.db import models
+
+from django.utils.translation import gettext_lazy as _
+
+
+class ShortLink(models.Model):
+    key = models.CharField(
+        primary_key=True,
+        max_length=32,
+        validators=[validators.validate_slug],
```

Comment:
You could also just use a SlugField https://docs.djangoproject.com/en/3.1/ref/models/fields/#slugfield